common/src/abstractions/userVerification.service.ts

```diff
@@ -6,4 +6,5 @@ export abstract class UserVerificationService {
     buildRequest: <T extends SecretVerificationRequest> (verification: Verification,
         requestClass?: new () => T, alreadyHashed?: boolean) => Promise<T>;
     verifyUser: (verification: Verification) => Promise<boolean>;
+    requestOTP: () => Promise<void>;
```

Comment:
Is this some way to avoid adding another dependency or? Seems to just be a wrapper to ApiService.

Reply:
It is just a wrapper, but it made sense to have request OTP and verify OTP in the same service. I don't really like calling `apiService` directly when you've got a more specific service handling that domain.

Reply:
No, I agree with encapsulating logic into services. Would be nice to remember to update references though. (In some ways we should stop directly using apiService and have it be used by other services in between but that's a topic for another day.)

Reply:
I'll update any references in the clients when I update jslib